File: 본문/python/Code_2_04_2.py
# 입력
N, X, Y = map(int, input().split())

# 답 구하기
cnt = 0
for i in range(1, N + 1):
	# mod 계산은 2.2절 참고
	if i % X == 0 or i % Y == 0:
		cnt += 1

# 출력
print(cnt)

File: 본문/cpp/Code_2_04_2.cpp
#include <iostream>
using namespace std;

int main() {
	// 입력
	int N, X, Y;
	cin >> N >> X >> Y;

	// 답 구하기
	int cnt = 0;
	for (int i = 1; i <= N; i++) {
		if (i % X == 0 || i % Y == 0) cnt++; // mod 계산은 2.2절 참고
	}

	// 출력
	cout << cnt << endl;
	return 0;
}